docs/sources/operator/api.md

```diff
@@ -0,0 +1,4618 @@
+---
+aliases:
+- /docs/agent/latest/operator/crd/
+title: CRD Reference
+weight: 500
+---
+# CRD Reference
```

Comment:
Can we spell this title out as we do for the "Custom Resource Quickstart"? So  "Custom Resource Definition Reference" or "Custom Resource Reference" if that's OK?

Reply:
I'm not sure if I understand. Why Quickstart? It's an extensive list of all possible options to configure the CRD of the agent operator.

Reply:
I think she means to change the title to "Custom Resource Reference" rather than "CRD Reference" :)

Reply:
Got it, will do it